### PR TITLE
shu-etal replication notebook

### PR DESCRIPTION
**Pull request by mohanadarafe**

### CODE REVIEW

File: shu-etal.ipynb [hunk 1]
```diff
@@ -0,0 +1,1516 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ed06fa75",
+   "metadata": {},
+   "source": [
+    "LivingPark notebooks use a *cache* directory to store analysis inputs and outputs. Inputs typically include PPMI Study Data and imaging data whereas outputs include processed images and other derivatives. Cache directories allow LivingPark notebooks to run in a few minutes as they reuse previously computed results. However, cache directories cannot be made public due to the PPMI Data Usage Agreement (DUA). Instead, they are stored on `login.bic.mni.mcgill.ca`, which requires a specific user name and password as well as a properly-configured DataLad installation in your account (see instructions [here](https://docs.google.com/document/d/1K7RzQjYC1O6tpSqe_c0CvxvJq38J10zNrDIdw20Yh_0/edit?usp=sharing)). In case you don't have access to the cache directory of this notebook, the next sections will download all the required imaging data from PPMI and recompute the results, which will take a few hours depending on your computer configuration. In the future, we will aim at storing this cache dataset on PPMI servers so that they can be accessed with a PPMI account.\n",
```
Comment (glatard) [inline shu-etal.ipynb]: The cache directory currently isn't functional, let's update this comment accordingly.

File: shu-etal.ipynb [hunk 2]
```diff
@@ -0,0 +1,1516 @@
...
+    "\n",
+    "utils.download_ppmi_metadata(required_files)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5bd67906",
+   "metadata": {},
+   "source": [
+    "We will also need file `MRI_info.csv` and `MDS_UPDRS_Part_III_clean_v0.1.csv` that are produced by another LivingPark notebook available at https://github.com/LivingPark-MRI/ppmi-MRI-metadata. This file contains a list of T1-weighted MRIs usable for VBM and a refined version of PPMI's UPDRS III data. "
```
Comment (glatard) [inline shu-etal.ipynb]: Add link to other notebook too.

File: shu-etal.ipynb [hunk 3]
```diff
@@ -0,0 +1,1516 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "429d9685",
+   "metadata": {},
+   "source": [
+    "## Hoehn & Yahr stage\n",
+    "\n",
+    "The first dataframe required will be extracted from `MDS_UPDRS_Part_III_clean.csv` with the objective to select a group of patients that are off medication on the day their exam was taken."
```
Comment (glatard) [inline shu-etal.ipynb]: not "on the day" but rather "during examination".

File: shu-etal.ipynb [hunk 4]
```diff
@@ -0,0 +1,1516 @@
...
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
```
Comment (glatard) [inline shu-etal.ipynb:427]: I'd be great to add a line to explain what this histogram is and to add axes titles.
Also, all the merging could be done in 1 cell.

File: shu-etal.ipynb [hunk 5]
```diff
@@ -0,0 +1,1516 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8a347dd1",
+   "metadata": {},
+   "source": [
+    "## Sanity checks\n",
+    "\n",
+    "Ensure that the cohort built does not have any defects (duplication, size, etc..)"
```
Comment (glatard) [inline shu-etal.ipynb]: List these issues explicitly in the text.

File: shu-etal.ipynb [hunk 6]
```diff
@@ -0,0 +1,1516 @@
...
+     ]
+    }
+   ],
+   "source": [
+    "print(f\"\\t\\t\\t\\t\\t| Stable set (n={len(stable)})\\t| Progression set (n={len(progr)})\" + os.linesep +\n",
+    "       f\"Age (years) \\t\\t\\t\\t| {round(stable['AGE_AT_VISIT'].mean(),2)} \\t\\t| {round(progr['AGE_AT_VISIT'].mean(),2)}\" + os.linesep + \n",
+    "       f\"Gender [male (n)] \\t\\t\\t| {len(stable[stable['SEX']==1])} \\t\\t\\t| {len(progr[progr['SEX']==1])}\" + os.linesep +\n",
+    "       f\"Hoehn & Yahr (n) \\t Stage 1 \\t| {len(stable[stable['NHY_x']=='1'])} \\t\\t\\t| {len(progr[progr['NHY_x']=='1'])}\" + os.linesep +\n",
+    "       f\"\\t\\t\\t Stage 2 \\t| {len(stable[stable['NHY_x']=='2'])} \\t\\t\\t| {len(progr[progr['NHY_x']=='2'])}\" + os.linesep +\n",
+    "       f\"\\t\\t\\t Stage 3 \\t| {len(stable[stable['NHY_x']=='3'])} \\t\\t\\t| {len(progr[progr['NHY_x']=='3'])}\"\n",
```
Comment (glatard) [inline shu-etal.ipynb]: Improve table formatting by putting the numbers in a dataframe. See cell 8 in https://github.com/LivingPark-MRI/scherfler-etal/blob/main/scherfler-etal.ipynb

File: shu-etal.ipynb [hunk 7]
```diff
@@ -0,0 +1,1516 @@
...
+    "<div class=\"alert alert-block alert-success\">\n",
+    "Shu, Zhen‐Yu, et al. <a href=\"https://onlinelibrary.wiley.com/doi/full/10.1002/mrm.28522?casa_token=Ab53WvMlODcAAAAA%3AXcgDLmq8egqW7uwd2g3jY9jIljhLu3VhIbvMWgbcfoWOxjO_9H7Arf91t2FBZDZ8E94Je4Wmrn0ZmkeZ\">Predicting the progression of Parkinson's disease using conventional MRI and machine learning: An application of radiomic biomarkers in whole‐brain white matter.</a> Magnetic Resonance in Medicine 85.3 (2021): 1611-1624.</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "078e819a",
+   "metadata": {},
+   "source": [
+    "This study recruited 72 patients with progressive PD and 72 patients with stable PD matched by age, sex and category of HYS. This following population from PPMI was used:\n",
```
Comment (glatard) [inline shu-etal.ipynb]: notebook intro should explain briefly what the replicated paper did and what was the main result

**Comment by glatard** [comment]
Hi @mohanadarafe, great work, I only added a few suggestions to add more explanation to the notebook text. It would be nice to read the notebook again and add as many explanations as possible to the text, to make sure it's readable by someone who would skip the code.

**Review by glatard — CHANGES_REQUESTED** [review]
Hi @mohanadarafe, great work! I only added a few suggestions to add more explanation to the notebook text. It would be nice to read the notebook again and add as many explanations as possible to the text, to make sure it's readable by someone who would skip the code. Also, please remove install.log from repo.

**Comment by mohanadarafe** [comment]
@glatard Thank you for the review, I updated the notebook accordingly. Let me know if it needs anything else!

**Pull request merged**